Solutions/IronNet IronDefense/Playbooks/IronNet_UpdateIronDefenseAlerts/azuredeploy.json

```diff
@@ -2,65 +2,69 @@
 	"$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
 	"contentVersion": "1.0.0.0",
 	"metadata": {
-		"comments": "Updates IronDefense alert workflow status and rating via IronAPI when the corresponding Sentinel Incident is updated.",
+		"comments": "Updates IronDefense alert workflow status and rating via IronAPI when the corresponding Microsoft Sentinel Incident is updated.",
 		"author": "IronNet"
 	},
 	"parameters": {
 		"IronApiUsername": {
-			"type": "String",
+			"type": "string",
+			"repeat": "false",
```

Comment:
Hello @ling-tx please can you tell if there is any need of this parameter "repeat":